docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.hero.scss

```diff
@@ -139,12 +140,11 @@
   }
 
   &__text {
-    background-color: darken($su-color-driftwood, 10%);
-    color: $hb-color--white;
```

Comment:
For future reference, colors like this white text are likely to be used across colorful/traditional/airy, so it might make sense to keep them here until we determine otherwise. 😄